pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1443,6 +1443,11 @@ func (og *operationGenerator) GenerateExpandVolumeFunc(
 			}
 
 			klog.Infof("ExpandVolume.UpdatePV succeeded for volume %s", util.GetPersistentVolumeClaimQualifiedName(pvc))
+		} else {
+			cancelled, err := cancelControlPlaneExpansion(pvc, pv, og.kubeClient)
+			if cancelled {
+				return err, err
```

Comment:
If cancel succeeded why return an error? Shouldn't it be the opposite?

Reply:
there are three scenarios that this covers:

1. if expansion cancel is successful - `err` will be automatically nil and hence we will be returning nil errors.
2. if cancelled is true but has errors that means - there might be an error updating PV/PVC and hence just api-server update should be retried and hence we have to return errors.
3. if cancelled is false then there was no attempt made to cancel anything and hence volume expansion should proceed as usual.

Reply:
Ack.